### PR TITLE
Add interface setCullingEnabled(), culling is disabled by default

### PR DESCRIPTION
**Pull request by dabingnn**

### CODE REVIEW

File: cocos/2d/CCSprite.cpp [hunk 1]
```diff
@@ -621,9 +622,15 @@ void Sprite::drawDebugData()
...
 
+void Sprite::setCullingEnabled(bool enabled)
+{
+    _cullingEnabled = enabled;
+}
+
 // Culling function from cocos2d-iphone CCSprite.m file
 bool Sprite::isInsideBounds() const
 {
+    if(!_cullingEnabled) return true;
```
Comment (minggo) [inline cocos/2d/CCSprite.cpp:633]: It is better to add a space line bellow.

Comment (dabingnn) [inline cocos/2d/CCSprite.cpp:633]: added
Thanks

File: cocos/2d/CCSprite.h [hunk 2]
```diff
@@ -426,6 +426,8 @@ class CC_DLL Sprite : public Node, public TextureProtocol
     virtual void setOpacityModifyRGB(bool modify) override;
     virtual bool isOpacityModifyRGB(void) const override;
     /// @}
+     //set culling enabled
+    void setCullingEnabled(bool enabled);
```
Comment (minggo) [inline cocos/2d/CCSprite.h:430]: Please describe why default value is false. What's the limitation of current culling.

**Comment by ricardoquesada** [comment]
why was this PR merged?

disabling auto-culling is not good, specially since we did many performance tests and we promoted the how fast v3.0 is.

we have to understand why auto-culling is not working for this case, but disabling it without further discussion is not a good idea.

**Comment by ricardoquesada** [comment]
BTW, usually I need 2 to 3 days to review and understand pull request. If the PR is urgent, please say so, I'll try to review it in the same day.

**Comment by ricardoquesada** [comment]
fixed here:
https://github.com/cocos2d/cocos2d-x/pull/6281

**Comment by minggo** [comment]
Sorry about that, @dabingnn said he had discussed it with you in email.

**Comment by dabingnn** [comment]
@ricardoquesada 
Sorry for my misunderstanding of your opinion.

**Comment by ricardoquesada** [comment]
Ok. no problem.

**Pull request merged**